yaksh/templates/yaksh/complete.html

```diff
@@ -32,5 +32,5 @@
 		<center><h2> Good bye! </h2></center>
 		<center><h4> {{message}} </h4></center>
 		<br><center><h4>You may now close the browser.</h4></center><br>
-	<center><a href="{{URL_ROOT}}/exam/" id="login_again"> Login Again </a></center>
+	<center><a href="{{URL_ROOT}}/exam/" id="go_to_home"> Go To Home </a></center>
```

Comment:
OK, how about just "Home" or "Dashboard", I just thought "Click here to return to Yaksh" seemed a bit too long for a Button label